packages/jest-editor-support/src/parsers/babylon_parser.js

```diff
@@ -9,8 +9,9 @@
 
 import {readFileSync} from 'fs';
 
-import {parse as babylonParse} from 'babylon';
+import {parse as babylonParse} from '@babel/parser';
 import {Expect, ItBlock} from './parser_nodes';
+// TODO: Change import to @babel/parser once types exist
 import type {File as BabylonFile} from 'babylon';
 
 export type BabylonParserResult = {
```

Comment:
@loganfsmyth @SimenB In Babylon, we were passing `plugins: ['*']`, but this was removed in Babel 7. What's the best way to infer the plugin list from the Babel config? Since I can't hardcode all of these plugins because flow vs typescript, etc.

Reply:
@orta thoughts?

Reply:
We've added a `babel.parse` that takes all of Babel's normal options and will load your plugins and such to decide how to parse, so that's the recommendation for this type of thing now.

Reply:
One thing to note is, like my other comment, this will return `null` if the file is `ignore`ed in the Babel config. Your call if you want to then fall back to something else of parse with `@babel/parser` with no `plugins` or what.